app/prices.js

```diff
@@ -10,6 +11,9 @@ const socket = require('lib/ptf-socket');
 
 const handlerManager = require('app/handler-manager');
 
+// Max age of a price in seconds
+const maxPriceAge = parseInt(process.env.MAX_PRICE_AGE) || 8 * 60 * 60;
```

Comment:
Maybe add a minimum?

Reply:
No need for a minimum, if you make MAX_PRICE_AGE be -1 then it will always check all items.

Reply:
Would it take significantly longer to start up if set to '-1' (2000+ listings assumption) @Nicklason ?

Reply:
Well, it would still "filter" the old prices even if you had -1 as the max age, I guess I will make it check for a minimum.